### DISCUSSION TITLE
How to specific type,Equity or ETF, of the ticker

### DISCUSSION THREAD

**Discussion by ebitoro** [dpguthrie/yahooquery#168] [Q&A]
For some symbols like AJXA, 
There is ETF with the same symbols. And ticker will always query EFT one, not the equity one.

**Comment by iukea1** [comment]
> For some symbols like AJXA, 
> There is ETF with the same symbols. And ticker will always query EFT one, not the equity one.
> 

Maybe see if it is on a different exchange.

I don't know if yahoo actually labels if a symbol is an ETF or not
